### PR TITLE
add handleCmds api

### PR DESCRIPTION
**Pull request by yjhjstz**
I try to add `handleCmds` API to achieve type any word to get response.

I test like `help`, `bt` ，that would be ok , but when I type 'v8 bt', It got error.

can you help ?

### CODE REVIEW

File: test/lldbdemo.js [hunk 1]
```diff
@@ -43,6 +43,7 @@ function my_listener(request, response) {
             llnode_module.loadDump(inputData.split("=")[1], process.env._);
             // Display thread stacks in a table
             var threads = llnode_module.getThreadCount();
+            console.log(llnode_module.handleCmds('plugin load /media/psf/Home/Work/llnode-api/out/Release/lib.target/llnode.so'));
```
Comment (yjhjstz) [inline test/lldbdemo.js]: try to load *.so, but got `this file does not represent a loadable dylib`.

**Comment by yjhjstz** [comment]
https://github.com/yjhjstz/llnode/commit/0127ddbf81f6253bc7f9cd515d2ec81fb1000090 

slove this.

**Comment by rnchamberlain** [comment]
Cool! It's a useful 'short-cut' approach to the API, having a function to issue any of the lldb/llnode line-mode commands and get the results back. So for example running 'v8 bt' rather than the 'getThread(), getFrame(), getSymbol() approach. It immediately gives access to a whole set of information, so in effect a JS wrapper for lldb/llnode. I'll build and have a play with it - thanks!

Incidently the diff looks a bit strange when I view it in github. Some changes I already have in my branch are showing up in the diff. For example the comment-out of a couple of fprintf calls at lines 30 and 38 in  src/llnode_api.cc, also some minor changes in test/lldbdemo.js.

**Comment by yjhjstz** [comment]
Sorry , a little bit behind. 
but I test 'v8' , that's ok. And test 'v8 bt', got error.
Can we  work together to fix this?

**Comment by yjhjstz** [comment]
another idea, use spawn and pipe achieve remote `repl` .
https://github.com/yjhjstz/llnode/commit/42635e5ef1b7915108c7abfe72f9a9c5b1daa3bf

**Comment by rnchamberlain** [comment]
I am seeing 'error: no such file' on the call to llnode_module.handleCmds('plugin load llnode.so'). So it probably can't find the llnode plugin .so. Note that my fork does not build the plugin, it builds a native module instead. So to run your command handler I think we need to change the project to build both the native module and the plugin. The base lldb commands like 'bt' don't need the plugin of course.

**Comment by rnchamberlain** [comment]
OK, so I built the plugin using 'gyp_llnode' and 'make' as per the original llnode project. Now the 'plugin load...' and 'v8' commands seem to work OK (well, I get some of the llnode help text appearing at least). But 'v8 bt' returns an error. Maybe we are doing something wrong loading the plugin that way. There is another class lldb::SBCommandPluginInterface, perhaps we should use that.

**Comment by yjhjstz** [comment]
Right now , I go through by using spawn , see https://github.com/yjhjstz/llnode/commit/ef998a22f353e38129366092f87e914afaaa24b5

**Comment by yjhjstz** [comment]
any progress? @rnchamberlain

**Comment by rnchamberlain** [comment]
@yjhjstz not had time in last couple of days, hope to get back on it soon :-) just wondering if having the same llnode code built into 2 binaries (one for the javascript native module and one for the plugin) is breaking something. it might be better to build your command handler as a separate project. i'll do some more digging anyway

**Comment by yjhjstz** [comment]
I refer to https://github.com/llvm-mirror/lldb/blob/master/examples/plugins/commands/fooplugin.cpp, 

this is a simple example that use `lldb::SBCommandPluginInterface`, so I confused.

**Comment by yjhjstz** [comment]
@rnchamberlain I have figure it out that `llnode`  forgot to set result status. Later I will send a pr . :)

**Comment by rnchamberlain** [comment]
@yjhjstz ah, well done, yes if I add this:    result.SetStatus(lldb::ReturnStatus::eReturnStatusSuccessFinishResult);

at the end of BacktraceCmd::DoExecute() in llnode.cc it works :-)

**Comment by yjhjstz** [comment]
Yes , in this way we just bind one api `handleCommands` to achieve access all commands. :)

**Comment by yjhjstz** [comment]
let's discuss here https://github.com/indutny/llnode/pull/20

**Pull request closed without merging**